CHANGELOG.md

```diff
@@ -3,12 +3,22 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
+
+## [2.2.1]
 ### Changed
-* Updated dependencies
+* Updated dependencies ([#246])
+* Erroring test reading symbolic link to dir ([#229])
```

Comment:
If you check this PR, you'll see it actually includes more than a failing test; it also includes a fix for that test.